xknx/knxip/connect_response.py

```diff
@@ -90,6 +68,79 @@ def __repr__(self) -> str:
             f'communication_channel="{self.communication_channel}" '
             f'status_code="{self.status_code}" '
             f'data_endpoint="{self.data_endpoint}" '
+            f'crd="{self.crd}" />'
+        )
+
+
+class ConnectResponseData:
+    """Representation of a KNX Connect Response Data block (CRD)."""
+
+    CRD_LENGTH = 2
+    CRD_TUNNEL_LENGTH = 4
+
+    def __init__(
+        self,
+        request_type: ConnectRequestType = ConnectRequestType.TUNNEL_CONNECTION,
+        identifier: int = 0,
```

Comment:
Since this is only used for tunnelling, I guess it always represents an `IndividualAddress`. Then we could just use that type instead of `int` here.

Reply:
Updated the PR.